cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Crossing the streams?! How dare you, @iloveagent57. I think all that's needed for that is an addition to the dict defined in `cms/celery.py`. That should do a text replacement on the value of POLICY_CHANGE_GRADES_ROUTING_KEY (`s/cms/lms/`) and get us where we want.

The test to confirm that worked is to load it all up on a sandbox, run the task, and confirm (via flower, I guess?) that it ran on an lms queue despite the signal coming from a cms process.